server/src/infra/auth_service.rs

```diff
@@ -314,6 +330,55 @@ where
     get_login_successful_response(&data, &name).await
 }
 
+async fn conventional_login<Backend>(
+    data: web::Data<AppState<Backend>>,
+    request: web::Json<login::ClientConventionalLoginRequest>,
+) -> HttpResponse
+where
+    Backend: TcpBackendHandler + BackendHandler + OpaqueHandler + 'static,
+{
+    let password = request.password.clone();
```

Comment:
I don't think you need this clone. You can write `let password = & request.password;`

Reply:
fixed